docs/v3/tutorials/airflow.mdx

```diff
@@ -0,0 +1,502 @@
+
+Prefect's model eliminates long-running sensor tasks, making workflows **more efficient, scalable, and event-driven**.
+
+### Airflow Hooks and Integrations
+Airflow provides hooks and operators for interacting with external systems (e.g., **JDBC, cloud services, databases**). In Prefect, these integrations are handled through [**Prefect Integrations**](/integrations/integrations) (e.g., `prefect-snowflake`, `prefect-gcp`, `prefect-dbt`) or by directly using the relevant **Python libraries** within tasks.  
+
+**Migrating Airflow Hooks to Prefect**  
+1. **Identify Airflow hooks** used in your DAGs (e.g., `PostgresHook`, `GoogleCloudStorageHook`).  
+2. **Replace them with equivalent Prefect integrations** or direct Python library calls.  
+   - Example: Instead of  
+     ```python
+        hook = PostgresHook(postgres_conn_id=my_conn_id)
+        engine = hook.get_sqlalchemy_engine()
+        session = sessionmaker(bind=engine)()
+     ```
+     Use Prefect Blocks for secure credential management:
+     ```python
+        from prefect_sqlalchemy import SqlAlchemyConnector
+
+        SqlAlchemyConnector.load("BLOCK_NAME-PLACEHOLDER")
+     ```
+3. **Use Prefect Blocks for secrets management**, similar to Airflow Connections, to separate credentials from code.
+
+**Replacing Airflow Operators with Prefect Tasks**
+- **Prefect tasks** can call any Python library, eliminating the need for custom Airflow operators.
+- Example: Instead of using a **BashOperator** to call an API via a shell script, install the necessary package in the flow's environment and call it directly in a task.
+
+Prefect's approach **removes unnecessary abstraction layers**, allowing direct access to the full Python ecosystem without Airflow-specific constraints.
+
+Basically: **anything done with a custom Airflow operator or hook can be replaced in Prefect with a task using the appropriate Python library.** Prefect removes Airflow's constraints, allowing direct use of the full Python ecosystem. For example, instead of using a **BashOperator** to call an API via a shell script, install the required package in your environment and call it directly from a task, eliminating unnecessary workarounds.
+
+### Observability
+
+#### State and logging
+**Task and Flow State Management**
+In Airflow, task states (`success`, `failed`, `skipped`, etc.) are stored in a metadata database and displayed in the Airflow UI's DAG run view. Prefect also tracks state for **each task and flow run**, but these states are managed by the **Prefect backend** (Prefect Server or Cloud API) and can be accessed via the **Prefect UI, API, or CLI**.
+
+After migration, similar visibility is available in Prefect's UI, where you can track which flows and tasks succeeded or failed. Prefect also includes additional state management features such as: 
+- **Flow cancellation** (`Cancelling` state).
```

Comment:
Kind of a nit but 'cancelling' is a temporary state - runs should get to a 'cancelled' state. I'd also say "flow run" cancellation.